src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -849,6 +869,33 @@ protected String createStepDescription(ExtensionContext context, final ItemType
 		return defaultValue;
 	}
 
+	protected String createStepMethodName(ExtensionContext context, final ItemType itemType) {
```

Comment:
Why do you need absolutely the same method twice?

Reply:
Made one method.Resolved